README.md

```diff
@@ -1,7 +1,5 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-<!-- add readme -->
-
 # Movie
 
```

Comment:
- [ ] OPTIONAL
Good job for making changes, however kindly make it a point to use the original _Pull Request_ for submitting the same projects after you implemented changes.